hw11/matrix_class.py

```diff
@@ -0,0 +1,56 @@
+class Matrix:
+    def __init__(self, matrix):
+        """
+        Конструктор класса
+        :param matrix: Двумерный список
+        """
+        self.matrix = matrix
+
+    def print_matrix(self):
+        """Выводим матрицу на экран"""
+        for row in self.matrix:
+            print(*row)
+
+    def __eq__(self, other):
+        """
+        Сравнение матриц путём сравнения.
+        :param other: Объект класса Matrix
+        :return: Boolean True/False
+        """
+        if self.matrix != other.matrix:
+            return False
+        return True
```

Comment:
Странная запись, почему не так
```suggestion
        if self.matrix == other.matrix:
            return True
        return False
```